src/models/db/accounts.js

```diff
@@ -30,7 +30,11 @@ function * _upsertAccount (account, options) {
     existingAccount.setData(account)
     yield existingAccount.save(options)
   } else {
-    yield Account.create(account, options)
+    if (!account.balance) {
+      yield Account.create(_.merge({}, account, {balance: 0}), options)
+    } else {
+      yield Account.create(account, options)
+    }
```

Comment:
This branch doesn't yet have that code. It's still using the old models: https://github.com/interledger/five-bells-ledger/blob/bb3c60570f5d3030fa0df3ecd6e5971a6c6e33e9/test/helpers/db.js#L66. So it's here: https://github.com/interledger/five-bells-shared/blob/511482962dfccd0c5c9b57a57734edc2faac4f39/lib/persistent-knex-model-mixin.js#L117.